discord/user/clientuser.py

```diff
@@ -2,64 +2,15 @@
 
 from typing import Optional
 
-from ..cache import UserCache
 from ..types.enums.locale import Locale
 from ..types.enums.userflags import UserFlags
 from ..types.userpayload import UserPayload
 from ..user.baseuser import BaseUser
 
 
 class ClientUser(BaseUser):
```

Comment:
Why is ClientUser almost completely deleted here?

Reply:
Because all of the common attributes of `ClientUser` and `User` was added to `BaseUser`

Reply:
Still unresolved